python/ray/data/dataset.py

```diff
@@ -3397,6 +3398,9 @@ def write_bigquery(
             dataset: The name of the dataset in the format of ``dataset_id.table_id``.
                 The dataset is created if it doesn't already exist. The table_id is
                 overwritten if it exists.
+            max_retry_cnt: The maximum number of retries that an individual block write
+                will be retried due to BigQuery rate limiting errors. This is not
+                related to Ray fault tolerance retries.
```

Comment:
https://docs.ray.io/en/latest/ray-contribute/docs.html#editorial-style

```suggestion
            max_retry_cnt: The maximum number of retries that an individual block write
                is retried due to BigQuery rate limiting errors. This isn't
                related to Ray fault tolerance retries.
```